### PR TITLE
no more roundstart liggers

### PR DESCRIPTION
**Pull request by GuppyLaxx**
:cl: optional name here
fix: fixed the roundstart race list
/:cl:

### CODE REVIEW

**Comment by ishitbyabullet** [comment]
finally we get some real chance

**Comment by Kraseo** [comment]
hhaha LIGGER i am so freakin ethnic XDDD

**Comment by LanCartwright** [comment]
upvoted

**Comment by qbmax32** [comment]
upvoted

**Comment by juanototo2** [comment]
upvoted

**Comment by davethwave** [comment]
In the first place just why...

**Comment by Xhuis** [comment]
Gittide's high tonight.

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	exotic_bloodtype = "L"
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
-
```
Comment (JJRcop) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: Could you re-add this part? It looks really ugly otherwise.

Comment (JJRcop) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: It looks like this right now:
https://github.com/tgstation/tgstation/blob/3edd031c86355ab0bbe5f0995aef96685f04d7f0/code/modules/mob/living/carbon/human/species_types/lizardpeople.dm#L20-L22

And that's really ugly looking, we always have a padding space between type definitions and procs, or even between procs, it just looks better that way.

**Comment by DoctoreMad** [comment]
>guy didnt like this so i had to make an empty line

You removed it yourself, mate.

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm (hunk 1 above)
Comment (JJRcop) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm]: Resolvez

**Comment by kevinz000** [comment]
please leave the dumb meme pr spam to goof.

**Comment by KorPhaeron** [comment]
This PR only stops xenobio from making lizards, test your garbage before opening a PR

**Pull request closed without merging**